### PR TITLE
Feature/fetch images

### LINKED ISSUE #24: Fetch images from Flickr API for Bird cards
- [ ] Create fetch call to retrieve images from Flickr API
- [ ] Create function to generate URL from Flickr response data
- [ ] Implement URLs to render images for each Bird card

### PR DESCRIPTION
**Pull request by LKessell**
# Pull Request Template
## Description

- Create a fetch call for images from Flickr API
- Fetch and render an image for each Bird card rendered

## Type of change
- [ ] Bug fix 
- [x] New feature
- [ ] Refactoring

## Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] My changes generate no new errors/warnings

### CODE REVIEW

**Comment by sydnerd** [comment]
Great job fetching the images and using a hook

File: src/components/App/App.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ import StatePicker from "../StatePicker/StatePicker";
 import BirdList from "../BirdList/BirdList";
 import { getBirds } from "../../apiCalls";
 import BirdDetails from "../BirdDetails/BirdDetails";
-import MyBirds from "../MyBirds/MyBirds";
```
Comment (k-atwhite) [inline src/components/App/App.js:9]: Is there a better way to order our imports?

File: src/components/Bird/Bird.js [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+  const [photos, setPhotos] = useState([]);
+
+  const createURL = (photo) => {
+    const photoUrl = 'https://live.staticflickr.com/' + photo.server + '/' +
+      photo.id + '_' + photo.secret + '_b.jpg';
+
+      return photoUrl;
+  }
+
+  getImages(scientificName)
```
Comment (k-atwhite) [inline src/components/Bird/Bird.js:18]: We need to style these gross boxes

File: src/components/App/App.js [hunk 3]
```diff
@@ -55,7 +54,7 @@ const App = () => {
             <BirdList birdData={regionBirds} />
           </Route>
           <Route path="/myBirds">
-            <MyBirds birdData={myBirds} />
```
Comment (k-atwhite) [inline src/components/App/App.js:58]: bye bye

**Review by k-atwhite — COMMENTED** [review]
Good job reusing that BirdList component

**Pull request merged**